test/guit/fetch.t

```diff
@@ -0,0 +1,18 @@
+Tests fetch command
+  $ mkdir simple
+  $ mkdir simple-clone
+  $ cd simple
+  $ git init
+  Initialized empty Git repository in $TESTCASE_ROOT/simple/.git/
+  $ git commit --allow-empty -q -m .
```

Comment:
Need `user.name` and `user.email` to be set before this.

Reply:
It seems that `dune runtest` changes my global config (see 6aaa6a4), I will try to find a way to locally set these values.

Reply:
`git config` without `--global` sets it per repository (you just have to run it after *every* `git init`.

Maybe it's time for a sandboxed dune test runner ;)